plugin.json

```diff
@@ -39,6 +39,12 @@
                 "type": "text",
                 "help_text": "Copy the **Client Secret** value (not ID) that was created on the App's **Certificates and Secrets** tab.",
                 "default": ""
+            },
+            {
+                "key": "EncryptionKey",
+                "display_name": "At Rest Encryption Key:",
+                "type": "generated",
+                "help_text": "The AES encryption key used to encrypt stored access tokens."
```

Comment:
Can we also add a word of caution in the `help_text` field which indicates that when this token is regenerated - all the previously stored access tokens will be invalidated and all the users will be required to re-authenticate.